packages/malloy/src/lang/parse-error-handler.ts

```diff
@@ -46,6 +46,7 @@ export class MalloyParserErrorHandler implements ANTLRErrorListener<Token> {
     const error: LogMessage = {
       message: msg,
       at: {url: this.translator.sourceURL, range},
+      severity: 'error',
```

Comment:
@mtoy-googly-moogly You mentioned you didn't want `severity` to be required in the logging functions. Is this okay, as this logging function takes a `LogMessage` directly? Or would you prefer that I change the signature of this function to either `log(message: string, at?: SourceLocation, severity = 'error')` or `log({ message, at, severity = 'error' }: { message: string; at?: SourceLocation; severity?: string })`